example/print-manager-v2-example.ts

```diff
@@ -80,53 +82,52 @@ const onScaleChange = event => {
  *
  */
 const registerPrintHandler = () => {
-  const printBtn = document.querySelector('button#print');
+  const printBtn: any = document.querySelector('button#print');
   printBtn.onclick = onPrintClick;
 };
 
 /**
  *
  */
 const onPrintClick = () => {
-  printProvider.print(true)
-    .catch(error => {
-      console.log('Error while printing: ' + error);
-    });
+  printProvider?.print(true)?.catch(error => {
+    console.log('Error while printing: ' + error);
+  });
 };
 
 /**
  *
  */
```

Comment:
not from you, but all empty comments can be removed in my opinion

Reply:
True, I suggest going over all that code when properly typing things.

Reply:
sounds good :+1: